firefox_desktop/views/desktop_daily_retention.view.lkml

```diff
@@ -0,0 +1,355 @@
+    sql: ${TABLE}.num_clients_dau_active_atleastonce_in_last_7_days ;;
+  }
+
+  dimension: num_clients_dau_on_day {
+    hidden: yes
+    sql: ${TABLE}.num_clients_dau_on_day ;;
+  }
+
+  dimension: num_clients_dau_repeat_first_month_users {
+    hidden: yes
+    sql: ${TABLE}.num_clients_dau_repeat_first_month_users ;;
+  }
+
+  dimension: num_clients_in_cohort {
+    hidden: yes
+    sql: ${TABLE}.num_clients_in_cohort ;;
+  }
+
+  dimension: num_clients_repeat_first_month_users {
+    hidden: yes
+    sql: ${TABLE}.num_clients_repeat_first_month_users ;;
+  }
+
+  # Custom
+
+  # Define measures
+  measure: cohort_measure {
+    label: "New Profiles (cohort)"
+    type:  sum
+    sql: (${TABLE}.num_clients_in_cohort) ;;
+  }
+
+  measure: num_clients_active_on_day_measure {
+    label: "clients - sent ping on day"
+    type: sum
+    sql:  ${TABLE}.num_clients_active_on_day ;;
+    hidden: yes
+  }
+
+  measure: num_clients_dau_on_day_measure {
+    label: "clients - qualified as DAU on day"
+    type: sum
+    sql:  ${TABLE}.num_clients_dau_on_day ;;
+    hidden: yes
+  }
+
+  measure: num_clients_active_atleastonce_in_last_7_days_measure {
+    label: "clients - sent ping at least once in last 7 days"
+    type: sum
+    sql:  ${TABLE}.num_clients_active_atleastonce_in_last_7_days ;;
+    hidden: yes
+  }
+
+  measure: num_clients_dau_active_atleastonce_in_last_7_days_measure {
+    label: "clients - DAU at least once in last 7 days"
+    type: sum
+    sql:  ${TABLE}.num_clients_dau_active_atleastonce_in_last_7_days ;;
+    hidden: yes
+  }
+
+  measure: num_clients_active_atleastonce_in_last_28_days_measure {
+    label: "clients - sent ping at least once in last 28 days"
+    type: sum
+    sql:  ${TABLE}.num_clients_active_atleastonce_in_last_28_days ;;
+    hidden: yes
+  }
+
+  measure: num_clients_dau_active_atleastonce_in_last_28_days_measure {
+    label: "clients - DAU at least once in last 28 days"
+    type: sum
+    sql:  ${TABLE}.num_clients_dau_active_atleastonce_in_last_28_days ;;
+    hidden: yes
+  }
+
+  measure: num_clients_active_in_period {
+    label: "clients - sent a ping during the retention period "
+    type: sum
+    sql:  CASE WHEN {% parameter retention_period_picker %} = 1 THEN ${TABLE}.num_clients_active_on_day WHEN {% parameter retention_period_picker %} = 7 THEN ${TABLE}.num_clients_active_atleastonce_in_last_7_days WHEN {% parameter retention_period_picker %} = 28 THEN ${TABLE}.num_clients_active_atleastonce_in_last_28_days ELSE 0 END;;
+  }
+
+  measure: num_clients_dau_in_period {
+    label: "clients - qualified as DAU during the retention period "
+    type: sum
+    sql:  CASE WHEN {% parameter retention_period_picker %} = 1 THEN ${TABLE}.num_clients_dau_on_day WHEN {% parameter retention_period_picker %} = 7 THEN ${TABLE}.num_clients_dau_active_atleastonce_in_last_7_days WHEN {% parameter retention_period_picker %} = 28 THEN ${TABLE}.num_clients_dau_active_atleastonce_in_last_28_days ELSE 0 END;;
```

Comment:
```suggestion
    sql:  CASE WHEN {% parameter retention_period_picker %} = 1 THEN ${TABLE}.num_clients_dau_on_day WHEN {% parameter retention_period_picker %} = 7 THEN ${TABLE}.num_clients_dau_active_atleastonce_in_last_7_days WHEN {% parameter retention_period_picker %} = 28 THEN ${TABLE}.num_clients_dau_active_atleastonce_in_last_28_days ELSE 0 END;;
    description: "The number of clients that qualified as DAU during this period. We qualify activity based on active_ticks, where the user must have moved a mouse or pressed keys for more than 5 seconds."
```

Again, I don't think I'm remembering this defn correctly

Reply:
Added a version of this description that included the > 0 uri part.